client/src/main/java/com/paypal/selion/platform/html/support/HtmlElementUtils.java

```diff
@@ -187,40 +187,41 @@ public static By resolveByType(String locator) {
      */
     public static By getFindElementType(String locator) {
         logger.entering(locator);
-        Preconditions.checkArgument(StringUtils.isNotBlank(locator), INVALID_LOCATOR_ERR_MSG);
+        String tempLocator = locator;
```

Comment:
Let's give this a better name (like `seleniumLocator`) and create this object after the `Precondition` where the assignment happens.